src/jobs/release-by-version.yml

```diff
@@ -0,0 +1,55 @@
+description: |
+  Publish a new GitHub Release for a repository by finding the version file and CHANGELOG.
+  This is intended for Python Repositories since we default to the version.py but other repository
+  use is allowed.
+
+executor: github
+
+parameters:
+  version-path:
+    description: |
+      File path to file containing the version. If not specified will default to version.py.
+      We will split the line matching version=x.y.z to get the version.
+    type: string
+    default: version.py
+  changelog-path:
+    description: File path to the changelog containing the description of the release.
+    type: string
+    default: CHANGELOG.md
+  file-path:
+    description: |
+      File directory to be uploaded in addition to the full repository.
+    type: string
+    default: ""
+  is-draft:
+    description: If true, then the release is a draft.
+    type: boolean
+    default: false
+  is-pre-release:
+    description: If true, then the release is a pre-release.
+    type: boolean
+    default: false
+
+steps:
+  - checkout
+  - run:
+      name: Provide error if awk is not installed.
+      command: |
```

Comment:
Same as above - do we need a file? If we do, can we use `/dev/null` instead?

Reply:
nope